build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.4.0-alpha7'
+        classpath 'com.android.tools.build:gradle:3.0.0-alpha4'
```

Comment:
I actually just downgraded to 2.3.3 which is the current stable version, beta versions expire after some amount of time (40 days?) and its too hard to keep up to date.